server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/repository/Repository.java

```diff
@@ -0,0 +1,60 @@
+package de.tum.in.www1.hephaestus.codereview.repository;
+
+import java.time.Instant;
+import java.util.List;
+
+import de.tum.in.www1.hephaestus.codereview.pullrequest.Pullrequest;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "repositories")
+@Getter
+@Setter
+@NoArgsConstructor
+public class Repository {
```

Comment:
Next to the properties you have listed I think the following are also interesting, potentially as a follow-up:
- `owner` / `owner_id` -> but we can do it as a followup, I think this would be a user object which is either an organization or a user AFAIK
- `pushed_at`
- `created_at`
- `updated_at`
- `homepage`
- `language`
- `default_branch`
- `open_issues_count`
- `stargazers_count`
- `watchers_count`
- `subscribers_count`
- `private`
- `archived`
- `disabled`
- `visibility`

Reply:
I added a few of these that felt like they could be useful for the leaderboard (commit f9af54a)